docs/preview/gha-runner-scale-set-controller/README.md

```diff
@@ -219,6 +219,25 @@ To fix this, you can either:
 
 ## Changelog
 
+### v0.4.0
+
+#### Major changes
+
```

Comment:
I think we should add a warning here that this release contains breaking changes and that end-users are advised to evaluate carefully before upgrading

Reply:
Added warning section. Let me know if I should phrase it differently :relaxed: